lms_latest.json

```
@@ -1,54 +1,70 @@
-						"host_default": 3483,
-						"label": "Control Channel"
-					}
-				},
-				"volumes": {
-					"/srv/squeezebox": {
-						"description": "Choose a Share for LMS configuration. E.g.: create a Share called lms-config for this purpose alone.",
-						"label": "Config Storage"
-					},
-					"/srv/music": {
-						"description": "Choose a Share for LMS content(your media). E.g.: create a Share called lms-music for this purpose alone. You can also assign other media Shares on the system after installation.",
-						"label": "Data Storage"
-					}
-				}
-			}
-		}
-	}
-}
+    "Lyrion Media Server - Latest Stable": {
+        "description": "Lyrion Media Server (formerly known as Logitech Media Server, Squeezebox, slimserver, etc.). Latest stable version. For future updates see more information icon after first installation.<p>Based on a official community docker image: <a href='https://hub.docker.com/r/lmscommunity/logitechmediaserver' target='_blank'>https://hub.docker.com/r/lmscommunity/logitechmediaserver</a>, available for amd64 and arm64 architecture.</p>",
+        "more_info": "<h4>Update to newer version</h4><p>Due to some current <a href='https://github.com/rockstor/rockstor-core/issues/2209' target='_blank'>limitations in rock-on networking</a>, the web UI port of this rock-on must be set as 9000 and will thus prevent the installation of other rock-ons needing this port. Alternatively, the web UI can be accessed manually at http://rockstor-ip:9000</p><p>After first installation: to update to a newer stable version, either uninstall/reinstall the Rockon or use the Watchtower Rockon to manage the update of the underlying Media Server image, as it does not automatically update itself during a stop/restart of the Rockon like some other Rockons do. Don't use the update option within the LMS!</p><p>The LMS configuration files will remain in place on the volumes that were specified during the initial installation</p>",
+        "website": "https://lyrion.org/",
+        "version": "2.4",
+        "ui": {
+            "slug": ""
+        },
+        "containers": {
+            "lms_latest": {
+                "image": "lmscommunity/logitechmediaserver",
+                "tag": "latest",
+                "launch_order": 1,
+                "opts": [
+                    [
+                        "--net",
+                        "host"
+                    ]
+                ],
+                "ports": {
+                    "9000": {
+                        "description": "WebUI port. This MUST be set to 9000",
+                        "host_default": 9000,
+                        "label": "WebUI port",
+                        "protocol": "tcp",
+                        "ui": true
+                    },
+                    "9090": {
+                        "description": "Command Line Interface (used for example by Android app Squeezer). Suggested Default: 9090",
```

```suggestion
                        "description": "Command Line Interface (used for example by Android app Squeezer). This MUST be set to 9090",
```
@Hooverdan96 Could you check my suggestion here: taken from: https://hub.docker.com/r/lmscommunity/logitechmediaserver

> Please note that both the http port 9000 and the cli port 9090 must always be mapped 1:1. 

There is more on this at the given reference though.

---

yes, good point. There are ways to changes these (when digging into the documentation a bit more), but it's like a three-way shuffle to do so. I will commit your suggestion.